pnpm-lock.yaml

```diff
@@ -9551,6 +10016,11 @@ packages:
     hasBin: true
     dev: false
 
+  /typescript@5.4.4:
```

Comment:
We shouldn't have any real additions to the lockfile here as the template has exactly the same dependencies as the base react template. Currently we are installing different versions of all of these packages can you reset back to main and install again?

For the eslint versions the monorepo moved to eslint 9 in https://github.com/palantir/osdk-ts/pull/230 which causes these peer dependencies errors for the templates (what the other PR comment was about):
```
 WARN  Issues with peer dependencies found
.
├─┬ @typescript-eslint/parser 7.7.1
│ └── ✕ unmet peer eslint@^8.56.0: found 9.1.1
├─┬ eslint-plugin-import 2.29.1
│ └── ✕ unmet peer eslint@"^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0 || ^8": found 9.1.1
├─┬ typescript-eslint 7.7.1
│ ├── ✕ unmet peer eslint@^8.56.0: found 9.1.1
│ └─┬ @typescript-eslint/eslint-plugin 7.7.1
│   ├── ✕ unmet peer eslint@^8.56.0: found 9.1.1
│   └─┬ @typescript-eslint/type-utils 7.7.1
│     ├── ✕ unmet peer eslint@^8.56.0: found 9.1.1
│     └─┬ @typescript-eslint/utils 7.7.1
│       └── ✕ unmet peer eslint@^8.56.0: found 9.1.1
└─┬ eslint-plugin-unused-imports 3.1.0
  └── ✕ unmet peer eslint@8: found 9.1.1
```

```
 WARN  Issues with peer dependencies found
.
├─┬ @typescript-eslint/parser 7.7.1
│ └── ✕ unmet peer eslint@^8.56.0: found 9.1.1
├─┬ eslint-plugin-import 2.29.1
│ └── ✕ unmet peer eslint@"^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0 || ^8": found 9.1.1
├─┬ typescript-eslint 7.7.1
│ ├── ✕ unmet peer eslint@^8.56.0: found 9.1.1
│ └─┬ @typescript-eslint/eslint-plugin 7.7.1
│   ├── ✕ unmet peer eslint@^8.56.0: found 9.1.1
│   └─┬ @typescript-eslint/type-utils 7.7.1
│     ├── ✕ unmet peer eslint@^8.56.0: found 9.1.1
│     └─┬ @typescript-eslint/utils 7.7.1
│       └── ✕ unmet peer eslint@^8.56.0: found 9.1.1
└─┬ eslint-plugin-unused-imports 3.1.0
  └── ✕ unmet peer eslint@8: found 9.1.1
```